client/app/billing/payment/method/add/views/legacyBillingAddress/controller.js

```diff
@@ -26,19 +24,8 @@ class LegacyBankAccountModel {
   }
 }
 
-export default class PaymentMethodAddLegacyBankAccountCtrl {
-  /* @ngInject */
-
-  constructor($state, currentUser) {
-    // dependencies injections
-    this.$state = $state;
-    this.currentUser = currentUser; // from app route resolve
-
-    // other attribute used in views
-    this.model = new LegacyBankAccountModel(currentUser.billingCountry);
-  }
-
+export default class PaymentMethodAddLegacyBillingAddressCtrl {
```

Comment:
Not yours but each concern should have its own file, controller in a `*.controller.js` and model in a `*.class.js`